nfc/CHANGELOG.rst

```diff
@@ -9,6 +9,14 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+master
+******
```

Comment:
```suggestion
Main branch
***********
```

Reply:
@maje-emb can you fix this and send another PR?